packages/cli/src/lib/history/history-command.ts

```diff
@@ -0,0 +1,68 @@
+import chalk from 'chalk';
+import { simpleGit } from 'simple-git';
+import { CommandModule } from 'yargs';
+import { HistoryOptions, history } from '@code-pushup/core';
+import { getCurrentBranchOrTag, safeCheckout } from '@code-pushup/utils';
+import { CLI_NAME } from '../constants';
+import { yargsOnlyPluginsOptionsDefinition } from '../implementation/only-plugins.options';
+import { HistoryCliOptions } from './history.model';
+import { yargsHistoryOptionsDefinition } from './history.options';
+
+export function yargsHistoryCommandObject() {
+  const command = 'history';
+  return {
+    command,
+    describe: 'Create history of commits',
+    builder: yargs => {
+      yargs.options({
+        ...yargsHistoryOptionsDefinition(),
+        ...yargsOnlyPluginsOptionsDefinition(),
+      });
+      yargs.group(
+        Object.keys(yargsHistoryOptionsDefinition()),
+        'History Options:',
+      );
+      return yargs;
+    },
+    handler: async args => {
+      // eslint-disable-next-line no-console
+      console.log(chalk.bold(CLI_NAME));
+      // eslint-disable-next-line no-console
+      console.log(chalk.gray(`Run ${command}`));
+
+      const currentBranch = await getCurrentBranchOrTag();
+      const {
+        targetBranch = currentBranch,
+        forceCleanStatus,
+        maxCount,
+        from,
+        to,
+        ...restOptions
+      } = args as unknown as HistoryCliOptions & HistoryOptions;
+
+      // determine history to walk
+      const git = simpleGit();
+      const log = await git.log({ maxCount, from, to });
+      const commitsToAudit = log.all
+        .map(({ hash }) => hash)
+        // crawl from oldest to newest
+        .reverse();
+
+      // run history logic
+      const reports: unknown[] = await history(
```

Comment:
Why cast it as `unknown[]`? The `history` function returns `string[]` promise.
```suggestion
      const reports = await history(
```